AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -416,7 +454,7 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
 
   ASCellNode *node = [_dataController nodeAtIndexPath:indexPath];
   
-  [_rangeController configureContentView:cell.contentView forCellNode:node];
+  [_rangeController configureContentView:cell.contentView forNode:node];
```

Comment:
Beyond naming, there is the touches forwarding and selection style properties. How about we just use cells and no need to worry about introducing indirection with another name for a node.